Source/Core/Common/Matrix.cpp

```diff
@@ -234,6 +234,76 @@ void Matrix44::Multiply(const Matrix44& a, const Matrix44& b, Matrix44* result)
   result->data = MatrixMultiply<4, 4, 4>(a.data, b.data);
 }
 
+// TODO: Perhaps this should return optional<>?
```

Comment:
This wasn't mentioned in the PR description, but this should also be resolved before this is merged.

Reply:
Thanks for taking a look lioncash!  This was copied from the OpenXR PR.  Looks like when that review was created there wasn't an `Inverted` function at all.  Now, in master there is a 3x3 `Inverted` function which just returns the value.  I'll go ahead and remove the comment since this matches that implementation.